src/components/QuranReader/ReadingView/ReadingPreviewSkeleton.module.scss

```diff
@@ -0,0 +1,17 @@
+@use "src/styles/breakpoints";
+@use "src/styles/utility";
+@use "src/styles/theme";
+
+.skeleton {
+  width: 100%;
+  min-width: calc(9.5 * var(--spacing-mega));
+  max-width: calc(15 * var(--font-size));
```

Comment:
will need to do follow up PR about the max-width when we implement the windowing